src/sentry/analytics/events/issue_alert_new_issue_sent_notification.py

```diff
@@ -0,0 +1,15 @@
+from sentry import analytics
+
+
+class IssueAlertNewIssueSentNotificationEvent(analytics.Event):
```

Comment:
i know analytics is the way to go here, but i wonder if we should just add a column to RuleFireHistory since everything here is identical

Reply:
we don't usually store things in the database that we aren't using in the app... but it feels a bit duplicate